sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeTasks.java

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.lro;
```

Comment:
why a special foldrer for lro?

Reply:
only healthcare and multiple tasks analysis are long-running operation, it is easier to find and better categorized.